tests/Aspire.Hosting.Tests/LaunchSettingsSerializerContextTests.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using Xunit;
+
+namespace Aspire.Hosting.Tests;
+public class LaunchSettingsSerializerContextTests
```

Comment:
nit: Add a blank line after the namespace

Reply:
Done